### PR TITLE
feat(core): expand rule configs for single run

### LINKED ISSUE #484: Additional rules properties in axe.run parameters
Would it be OK if I added more configurable properties to the rules property in axe.run's options parameter?  Right now I think the only property customizable on a per-run basis is 'enabled.'  This would be useful with WebDriver integrations, since we'd be able to configure a rule's selector without having to maintain the state of the axe object.

### PR DESCRIPTION
**Pull request by philipjia123**
Bring rule property customization in axe.run to parity with axe.configure

Closes #484

### CODE REVIEW

**Comment by WilcoFiers** [comment]
Please make sure the tests pass first.

**Comment by philipjia123** [comment]
D'oh, sorry about that

File: lib/core/base/rule.js [hunk 1]
```diff
@@ -325,10 +338,6 @@ Rule.prototype.configure = function (spec) {
 	}
 
 	if (spec.hasOwnProperty('matches')) {
-		if (typeof spec.matches === 'string') {
-			this.matches = new Function('return ' + spec.matches + ';')();
-		} else {
-			this.matches = spec.matches;
-		}
+		this.matches = createExecutionContext(spec.matches);
```
Comment (WilcoFiers) [inline lib/core/base/rule.js]: Good catch!

File: test/core/base/audit.js [hunk 2]
```diff
@@ -501,6 +501,84 @@ describe('Audit', function () {
 				done();
 			}, isNotCalled);
 		});
+		it('should use selectors for rules specified by the options', function (done) {
+			fixture.innerHTML = '<blink>BAD BLINKY ELEMENT</blink>';
+
+			a.run({ include: [axe.utils.getFlattenedTree(fixture)[0]] }, {
```
Comment (WilcoFiers) [inline test/core/base/audit.js]: I think you should just be able to pass in the fixture.

**Comment by WilcoFiers** [comment]
@philipjia123 Any updates on those tests / docs?

**Comment by marcysutton** [comment]
Hi @philipjia123, we're still waiting on tests and docs for this PR. Are you able to contribute those soon?

**Comment by marcysutton** [comment]
Closing due to inactivity. Please reopen if you want to continue @philipjia123!

**Pull request closed without merging**